### DISCUSSION TITLE
When using table in object brick whole pimcore crashes

### DISCUSSION THREAD

**Discussion by jdevries14** [pimcore/pimcore#10396] [Q&A / Support] [answered]
<img width="1137" alt="Screenshot 2021-09-23 at 12 05 50" src="https://user-images.githubusercontent.com/44669189/134489450-2f2fc003-a8af-4600-9480-7172a29d3ba7.png">

This is my current object brick, and when selecting the object brick in my class it works fine, look:

<img width="1559" alt="Screenshot 2021-09-23 at 12 06 40" src="https://user-images.githubusercontent.com/44669189/134489556-bbe5ba3b-96c8-44c0-b5b9-d9849d708f13.png">

But when I add a table to my object brick and give it a name without any settings as following:
<img width="933" alt="Screenshot 2021-09-23 at 12 07 24" src="https://user-images.githubusercontent.com/44669189/134489644-e94aaae9-c100-4f3b-a507-82f486454b97.png">

Then when I choose this object brick, it crashes and I can't use Pimcore anymore. Also it doesn't give an exception or so.
Also all my fields etc disappear as following:

<img width="722" alt="Screenshot 2021-09-23 at 12 07 59" src="https://user-images.githubusercontent.com/44669189/134489702-4650b9ec-bd69-481d-95c7-741db22146e4.png">

I do get a console error:

![Screenshot 2021-09-23 at 12 08 52](https://user-images.githubusercontent.com/44669189/134489807-980a45bd-c7ed-49a5-a236-77428a067e8b.png)

**Comment by weisswurstkanone** [comment, marked as answer]
regression of https://github.com/pimcore/pimcore/pull/10290

**Reply by dvesh3** [reply]
Fixed here https://github.com/pimcore/pimcore/pull/10400
